tests/integration/tests/test_smoke.py

```diff
@@ -54,3 +55,39 @@ def test_smoke(instances: List[harness.Instance]):
             ["cat", f"/var/snap/k8s/common/args/{service}"], capture_output=True
         )
         assert value in args.stdout.decode()
+
+
+def test_capi_auth(session_instance: harness.Instance):
```

Comment:
please don't use the session_instance here. This instance is meant to test the features and should not alter configuration.

Just move that test into the above smoke test and use the instance there.